Volume/Views/MainView/HomeList.swift

```diff
@@ -6,25 +6,49 @@
 //  Copyright © 2020 Cornell AppDev. All rights reserved.
 //
 
+import Combine
 import SwiftUI
 
 struct HomeList: View {
+    @State private var trendingArticles: [Article] = []
+    @State private var followingArticles: [Article] = []
+    @State private var otherArticles: [Article] = []
+    @State private var cancellableQuery: Cancellable?
+    
+    private func fetch() {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd"
+        guard let date = Calendar.current.date(byAdding: .day, value: -7, to: Date()) else { return }
+        let dateString = formatter.string(from: date)
+        let query = GetHomeArticlesQuery(since: dateString)
```

Comment:
Can we add this `dateString` to the Date extension as well (as a function or computed var)